src/application/models/v3/Patient_model.php

```diff
@@ -19,9 +19,9 @@ public function add($patient_integrated)
     public function validate($patient_integrated) {
         $this->load->helper('data_validation');
 
-        if (empty($patient_integrated['id_user_integrated']))
```

Comment:
Why did you remove this validation

Reply:
Make id_user_integrated column as required = false , following ticket requirement